flixel/FlxCamera.hx

```diff
@@ -635,6 +635,11 @@ class FlxCamera extends FlxBasic
 			itemToReturn = new FlxDrawItem();
 		}
 
+		if (graphic.wasDestroyed) {
+			// Throw an exception here, because throwing in `FlxDrawQuadsItem.render()` is not helpful.
+			throw 'Attempted to queue invalid FlxDrawItem, did you dispose of a cached sprite?';
```

Comment:
i want to avoid the word dispose, and just use the word "destroy" consistently. since that's the function they are likely calling